bin/preview-deploy/aws.sh

```diff
@@ -115,7 +115,7 @@ function createNewInstance() {
     --subnet-id "$AWS_SUBNET" \
     --tag-specification "ResourceType=instance,Tags=[{Key=Name,Value=eAPD PR $PR_NUM},{Key=environment,Value=preview},{Key=github-pr,Value=${PR_NUM}}]" \
     --user-data file://aws.user-data.sh \
-    --key-name eapd_bbrooks \
+    --key-name radavis \
```

Comment:
should we remove both of these keys?

Reply:
This line is going to change based on whomever needs to ssh into the preview server for a given branch.

The keys are stored on AWS. Specify which public key to copy onto the ec2 server, here.

Reply:
Ideally we would remove access to the backend servers for normal operations and only deploy them out when we need access